instaclone/src/Nav/Nav.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import * as S from './Navs'; 
+
+//img
+ const instalog = '../../../img/nav/InstagramTextLogo.svg';
+ const home = '../../../img/nav/HomeIcon.svg';
+ const search = '../../../img/nav/Search.svg';
+ const look = '../../../img/nav/Compass.svg';
+ const li = '../../../img/nav/Reels.svg';
+ const send = '../../../img/nav/Message.svg';
+ const Alarm = '../../../img/nav/Alarm.svg';
+ const the = '../../../img/nav/Plus.svg';
+ const thelook = '../../../img/nav/Burger.svg';
```

Comment:
```suggestion
import Instalog from "../../../img/nav/InstagramTextLogo.svg";

<img src={Instalog} />
```
이렇게 모듈로 써도 좋을 것 같습니다

Reply:
넵 참고하도록 하겠습니다

Reply:
그리고 대문자로 시작했으면 다 대문자로, 소문자로 했으면 다 소문자로 하는 식으로 통일을 시키면 더 좋을 것 같습니다!